code/modules/mob/living/carbon/human/species/station/alraune.dm

```diff
@@ -88,6 +90,17 @@
 		O_EYES =     /obj/item/organ/internal/eyes/alraune,
 		)
 
+/datum/species/alraune/equip_survival_gear(var/mob/living/carbon/human/H)
+	..()
+
+	var/implanter = /obj/item/weapon/implanter/reagent_generator/fruit_implant
+	var/obj/item/weapon/implanter/reagent_generator/fruit_implant/implant = new implanter
+
+	if(H.backbag == 1)
+		H.equip_to_slot_or_del(implant, slot_r_hand)
+	else
+		H.equip_to_slot_or_del(implant, slot_in_backpack)
```

Comment:
Why do you hand them a literal implanter instead of just implanting them?

Reply:
Is there a possible way to make it so they're implanted at round start & it pops up a message to select their fruit? If so, is there any examples in the code of something similar to that?

Reply:
The loadout system attempts to implant you at roundstart if you take things like the tracking implants

Reply:
If I recall correctly, Tracking Implants still just appear on the ground?

Reply:
It makes a copy, for some reason. There's still an implant added to the mob, though.